home/models.py

```diff
@@ -59,6 +58,7 @@ class RSSImport(models.Model):
     def __str__(self):
         return self.rss_title_en
 
+
```

Comment:
It keeps happening to me in Opin, that the when I try to commit, the linter complains about files, I never touched. So I just fix them. Should I check for that before I start working?